recipe/meta.yaml

```diff
@@ -10,34 +10,64 @@ source:
   sha256: 099d86f7c79c51a2caaf7c8598cc56e7b0a57c11b5918ddf699e82380735b6b7
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
+  skip: true  # [py<38 or s390x]
 
 requirements:
   host:
-    - python >=3.8,<4.0
+    - python 
     - hatchling
     - pip
   run:
-    - python >=3.8,<4.0
+    - python
     - faker
-    - typing-extensions
+    - typing-extensions >=4.6.0
```

Comment:
Would it make sense to add the pinned optional dependencies to to a `run_constrained` section here ?
- sqlalchemy>=1.4.29
- odmantic<1.0.0
- attrs>=22.2.0
- email-validator>=2.0.0 (from pydantic[email])

Reply:
I can add them all except:
- `email-validator 2.0.0` our highest version is `1.2.1`
- `odmantic` which we don't have

Reply:
I don't think we need to have them to add them to `run_constrained`.